src/components/ui/WorkExperience/index.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React from 'react'
+
+import InlineEditor from '../InlineEditor'
+import type { EditorProps, PresenterProps } from '../InlineEditor'
+import type { Experience } from '../../../utils/types'
+
+import WorkExperienceEntryPresenter from './WorkExperienceEntryPresenter'
```

Comment:
We've been using this pattern to import things, as separating with blank line by kind: components, styles, utils, types, etc.

```suggestion
import InlineEditor from '../InlineEditor'
import WorkExperienceEntryPresenter from './WorkExperienceEntryPresenter'

import type { EditorProps, PresenterProps } from '../InlineEditor'
import type { Experience } from '../../../utils/types'
```